borg/archive.py

```diff
@@ -180,7 +180,7 @@ def _load_meta(self, id):
     def load(self, id):
         self.id = id
         self.metadata = self._load_meta(self.id)
-        decode_dict(self.metadata, (b'name', b'hostname', b'username', b'time', b'time_end'))
+        decode_dict(self.metadata, (b'name', b'comment', b'hostname', b'username', b'time', b'time_end'))
```

Comment:
"Tagging" for some (future) cleanup: Make a METADATA_KEYS (or something like that) as we have one for the item keys (ITEM_KEYS), so that we can de-duplicate all these snippets (=> don't repeat yourself).